data/adiabatic-conditions/ascii-data/simple_test.txt

```diff
@@ -0,0 +1,7 @@
+# Test data for ascii data reference state
```

Comment:
I was also surprised to see test data in /data/. Because the data is not at all sensible, I would would to move this into tests. I can make a PR for the other files.